packages/Tag/src/Tag.tsx

```diff
@@ -21,44 +26,92 @@ export type Appearance =
 export interface TagProps extends React.ComponentProps<'div'> {
   /** The different appearances of the Tag */
   appearance?: Appearance;
-  /** Add a data-test tag for automated tests */
-  dataTest?: string;
   /** The content of the Tag */
   children: React.ReactNode;
   /** Add a specific class to the Tag */
   className?: string;
+  /** Add a colored square instead of an image or an icon */
+  color?: string;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
   /** Render the close button */
   dismissible?: boolean;
   /** The icon used at the beginning of the Tag */
-  icon?: React.ReactNode;
+  icon?: React.ReactElement;
+  /** The id of the tag */
+  id?: string;
+  /** Add an error style to the tag */
+  hasError?: boolean;
+  /** Add a warning icon to the tag. Overrides any image, color or icon */
+  isWarning?: boolean;
```

Comment:
It seems that this props is never used in the portal, in my opinion you can remove it